docker-entrypoint.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Exit immediately if a command exits with a non-zero status.
+set -e
+
+# Trace execution
+[[ "${DEBUG}" ]] && set -x
+
+# Set default key (not actually used)
```

Comment:
ok this anwers my previous comment about the secret key

Reply:
> is it a convention to name the root folder "django" ?

It's common tho not really a convention. Could have named it `river` or `src` but `api` is part of `river` and `src` is more conventional for distributed packages.